sos/report/plugins/gluster.py

```diff
@@ -103,8 +103,8 @@ def setup(self):
                     self.soslog.info("could not send SIGUSR1 to glusterfs/"
                                      "glusterd processes")
             else:
-                self.soslog.warning("Unable to generate statedumps, no such "
-                                    "directory: %s" % self.statedump_dir)
+                self.soslog.warning(f"Unable to generate statedumps, no such "
+                                    f"directory: {self.statedump_dir}")
```

Comment:
`W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)`

So replaced with f-string here.